bkg/v2/BKG_v2.0.0-Beta-2.yaml

```diff
@@ -4524,14 +4524,144 @@ components:
     # Document Parties
     ##################
 
+    PartyAddress1:
+      type: object
+      title: Address (with optional properties)
+      description: |
+        An object for storing address related information
+      properties:
+        street:
+          type: string
+          maxLength: 100
+          description: The name of the street of the party’s address.
+          example: Ruijggoordweg
+        streetNumber:
+          type: string
+          maxLength: 50
+          description: The number of the street of the party’s address.
+          example: '100'
+        floor:
+          type: string
+          pattern: ^\S(?:.*\S)?$
+          maxLength: 50
+          description: |
+            The floor of the party’s street number.
+          example: N/A
```

Comment:
I do not think `N/A` is a correct example here. It is not a valid floor designation. Ideally, it would use a proper floor or have the example be omitted (not sure how to do that in a way that does not trigger an auto-generated example that is even worse)

Reply:
Agree - fixed